server/tm_cmds.go

```diff
@@ -72,7 +72,7 @@ func UnsafeResetAllCmd(ctx *Context) *cobra.Command {
 		Short: "Reset blockchain database, priv_validator.json file, and the logger",
 		RunE: func(cmd *cobra.Command, args []string) error {
 			cfg := ctx.Config
-			tcmd.ResetAll(cfg.DBDir(), cfg.PrivValidatorFile(), ctx.Logger)
+			tcmd.ResetAll(cfg.DBDir(), cfg.PrivValidatorFile(), cfg.P2P.AddrBookFile(), ctx.Logger)
```

Comment:
The parameters are backwards for privValidator and addrbook. Causes priv_validator.json to be deleted. See #1380

Reply:
OMG I'm sorry. 

I had a feeling this "Delete AddrBook on UnsafeResetAll" was going to cause a problem somewhere.

This stuff is called UNSAFE for a reason!

Reply:
Maybe we should do something safer like pass in a config directly or anything to not just pass two filename strings that are treated differently. Yikes